heudiconv/cli/run.py

```diff
@@ -98,9 +103,20 @@ def process_extra_commands(outdir, args):
     return
 
 
+def help_bids():
```

Comment:
This can be added to the argument's `desc` field instead

Reply:
I added bids options information to the `help` field (as far as I can tell `desc` is only parser level).